SIMS/SIMS/SIMS.csproj

```diff
@@ -7,4 +7,18 @@
         <Nullable>enable</Nullable>
     </PropertyGroup>
 
+    <ItemGroup>
+        <PackageReference Include="Microsoft.Data.SqlClient" Version="5.2.0"/>
```

Comment:
where do you use this Microsoft.Data.SqlClient ?

Reply:
You are right, I haven`t used it, it seems it was added by mistake